GrandCentralBoardTests/BonusWidgetSnapshotTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  BonusWidgetSnapshotTests.swift
+//  GrandCentralBoard
+//
+//  Created by krris on 23/05/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import FBSnapshotTestCase
+import GCBCore
+import GCBUtilities
+@testable import GrandCentralBoard
+
+private final class PeopleWithBonusesProvider: PeopleWithBonusesProviding {
+    let shouldFailRequest: Bool
+
+    init(shouldFailRequest: Bool) {
+        self.shouldFailRequest = shouldFailRequest
+    }
+
+    private func fetchPeopleWithBonuses(completionBlock: (Result<[Person]>) -> Void) {
+        if shouldFailRequest {
+            return completionBlock(Result.Failure(ErrorWithMessage(message: "error_message")))
+        } else {
+            let people: [Person] = []
```

Comment:
Yes, but it can be hard to test, because:
1. We have to wait for SpriteKit scene to be loaded.
2. Bubbles are moving randomly. Screenshots in 99.9% will be different.